common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -78,10 +79,11 @@
 	private TimeoutPolicy timeoutPolicy = TimeoutPolicy.TIME_OUT_WF;
 
 	@ProtoField(id = 8)
-	private RetryLogic retryLogic = RetryLogic.FIXED;
+	private RetryLogic.RetryLogicPolicy retryLogicPolicy = RetryLogic.RetryLogicPolicy.FIXED;
```

Comment:
This would break backward compatibility in both proto (field id with a different type) and json (field name is different) serde. 

Let's keep the enum as-is within this class and add the new values to it. This enum can then be referenced from WorkflowTask as well.

Reply:
Done